shell/platform/embedder/embedder.cc

```diff
@@ -1710,3 +1722,27 @@ FlutterEngineResult FlutterEnginePostDartObject(
   typed_data_finalizer.Release();
   return kSuccess;
 }
+
+FLUTTER_EXPORT
+FlutterEngineResult FlutterEngineNotifyLowMemoryWarning(
```

Comment:
Consider also calling Dart_NotifyLowMemory.

Reply:
I did not know that was a think. I'll wire up in another patch. Thanks.

Reply:
https://github.com/flutter/engine/pull/14517/files